### DISCUSSION TITLE
Function calling is supported in the latest model released by Open AI, does SK plan to adapt it

### DISCUSSION THREAD

**Discussion by unsung189** [microsoft/semantic-kernel#1477] [Q&A] [answered]
Function calling is supported in the latest model released by Open AI, does SK plan to adapt it

https://openai.com/blog/function-calling-and-other-api-updates

**Comment by craigomatic** [comment, marked as answer]
Yes, see https://github.com/microsoft/semantic-kernel/issues/1450
